neurogym/wrappers/monitor.py

```diff
@@ -99,27 +98,36 @@ def step(self, action):
 
             # save data
             save = False
-            save = self.t >= self.sv_per if self.sv_stp == "timestep" else self.num_tr % self.sv_per == 0
+            save = (
+                self.t >= self.conf.monitor.plot.interval
+                if self.conf.monitor.trigger == "timestep"
+                else self.num_tr % self.conf.monitor.plot.interval == 0
+            )
             if save:
```

Comment:
this is quite confusing to me. Why is save defined twice? And why is it defined at all, rather than putting the condition directly in the if statement?

I realize that this was already the case before you touched the code, but maybe nice to fix it up while touching this part of the code.

Reply:
This is an artefact of the stacked PR, this part is completely overhauled in #146 because I'm using the `conf` object instead of all of these arguments..